models/Recipe.model.js

```diff
@@ -3,8 +3,16 @@ const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
   // TODO: write the schema
-});
+  title: {type: String, required: true, unique: true},
+  level: {type: String, enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef']},
+  ingredients: [String],
+  cuisine: {type: String, required: true},
+  dishType: {enum: ['breakfast', 'main_course', 'soup', 'snack', 'drink', 'dessert', 'other']},
+  image: {type: String, default: 'https://honest-food.net/wp-content/uploads/2013/12/portuguese-feijoada-320x320.jpg'},
+  duration: {type: Number, min: 0},
+  creator: {type: String},
+  created: {type: Date, default: new Date()}
 
-const Recipe = mongoose.model('Recipe', recipeSchema);
+});
 
-module.exports = Recipe;
+const Recipe = mongoose.model('Recipe', recipeSchema);
```

Comment:
remember to export your module; a way to do it in one sentence could be: module.exports = mongoose.model('Recipe', recipeSchema);